slice.go

```diff
@@ -7,18 +7,25 @@ import (
 	"strings"
 )
 
+const (
+	ENV_SEPARATOR     rune	= ','
```

Comment:
Could you unexport this and rename it to something like `sliceEnvSeparator` so it's clearer.
Same for below, unexport and rename it to something like `sliceDefaultSeparator`.